uaclient/cli.py

```diff
@@ -170,6 +169,12 @@ def action_enable(args, cfg):
 
     @return: 0 on success, 1 otherwise
     """
+    if cfg.is_attached and os.getuid() == 0:
+        # Refresh contracts prior to enable
+        contract_token = cfg.read_cache('contract-token')
```

Comment:
This logic is replicated in `action_refresh()` below; should it move in to a helper?

(Should we at least be debug logging that we're performing this?)

Reply:
read_cache does a util.load_file which adds a debug log entry about the file read. That said, I had wanted to drop the load_file debug logs as they are super noisy. So, maybe it's worth putting the log here instead.

Reply:
added.

Reply:
Added a decorator for this check  and applied it to detach, enable disable operations.